src/meta/src/dashboard/mod.rs

```diff
@@ -360,6 +361,24 @@ pub(super) mod handlers {
 
         Ok(report)
     }
+
+    pub async fn get_back_pressure(
+        // Path(worker_id): Path<WorkerId>,
+        Extension(srv): Extension<Service>,
+    ) -> Result<Json<GetBackPressureResponse>> {
+        let back_pressure_infos = match &srv.metadata_manager {
+            MetadataManager::V1(mgr) => {
```

Comment:
We can list all streaming worker nodes directly here by metadata manager, then we can call the RPC here and avoid to add interface for worker manager and controller.